lib/routes/veeam/utils.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * Decodes an URI and return the result.
+ * Do the same decoding than in S3 server
+ * @param {string} uri - uri to decode
+ * @returns {string} -
+ */
+function _decodeURI(uri) {
+    return decodeURIComponent(uri.replace(/\+/g, ' '));
+}
+
+const validPath = '.system-d26a9498-cb7c-4a87-a44a-8ae204f5ba6c/';
```

Comment:
I understand but I feel like we shouldn't mix custom route and standard configuration... And at some point, it may even be set at the kubernetes level... I'm fine putting it in constants.js but for now I do not really see a benefit of doing so. Please feel free to challenge my point of view